core/src/main/java/apoc/nodes/DeleteAndReconnectConfig.java

```diff
@@ -0,0 +1,34 @@
+package apoc.nodes;
+
+import apoc.util.Util;
+import org.neo4j.graphdb.Relationship;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class DeleteAndReconnectConfig {
+
+    private final boolean attachStartRel;
+    private final List<Relationship> relsToAttach;
+    private final List<String> relTypesToAttach;
+
+    public DeleteAndReconnectConfig(Map<String, Object> config) {
+        if (config == null) config = Collections.emptyMap();
+        this.attachStartRel = Util.toBoolean(config.getOrDefault("attachStartRel", true));
```

Comment:
io questo lo modificherei con `relationshipSelectionStrategy` con tre valori:

* `START`
* `END`
* `MERGE`
In quest'ultima se nel tuo test dove hai `(f:One)-[:ALPHA {a:'b'}]->(b:Two)-[:BETA {c:'d', e:'f'}]->(c:Three)-[:GAMMA]->(d:Four)-[:DELTA {aa: 'bb', cc: 'dd', ee: 'ff'}]->(e:Five {foo: 'bar', baz: 'baa'})` ed elimini `Two` diventa:

`(f:One)-[:ALPHA_BETA {a:'b',c:'d', e:'f'}]->(c:Three)-[:GAMMA]->(d:Four)-[:DELTA {aa: 'bb', cc: 'dd', ee: 'ff'}]->(e:Five {foo: 'bar', baz: 'baa'})`